extension/src/experiments/index.ts

```diff
@@ -169,13 +168,12 @@ export class Experiments extends BaseRepository<TableData> {
     return this.data.managedUpdate()
   }
 
-  public async setState(data: ExpShowOutput) {
+  public async setState({ expShow, gitLog, rowOrder }: ExperimentsOutput) {
```

Comment:
[I] `getNumCommits` should come out of `WebviewMessages` and called in `ExperimentsData` as well.

Reply:
I've got an item on the board to follow up and do this but will do it next.